### DISCUSSION TITLE
Upgrade to 1.10.0 from 1.9.1 procedure

### DISCUSSION THREAD

**Discussion by DoomDoomGir** [thehighnibble/thehighnibble.github.io#124] [Q&A] [answered]
If someone could confirm this procedure for me I would appreciate it.

I'm preparing to upgrade from 1.9.1 to 1.10.0.  I've read the docs on the website and I've read the conversations on this forum.  Here is my summary of how it should work.

1. Power on machine with 1.9.1 uSD card installed.
2. Open web interface and drag firmware file imsaisim_esp32.bin from the 1.10.0 FIRMWARE folder to the SYS: icon on the web interface.
3. Wait for update successful message.
4. Power off the machine.
5. Create a new uSD card from the 1.10.0 IMAGE folder.
6. Make any config changes needed for my environment.
7. Insert newly created uSD card.
8. Power the machine on.
9. System should start up using new firmware and new web interface should be displayed.

I think that should work.  Anyone else agree?  Thanks so much for your help.

Thanks,
Randy

**Comment by DoomDoomGir** [comment, marked as answer]
Never mind, it must have been a stupid question.  I followed my procedure and it worked fine.

**Comment by nbreeden2** [comment]
Glad it worked.
